### ISSUE TITLE
Still not reproducible when built on target

### ISSUE THREAD

**Issue by jrick** [jrick/release#2]
Thought the reproducibility issues were settled after #1 but if comparing two builds for the same target, one built from the target and the other not, the bits are different.

The hexdump diff for an OpenBSD dcrctl compiled on OpenBSD and macOS is too large for a GitHub issue.

When inspecting the executables with ldd, the one built from macOS is static, while the one built from OpenBSD links to:

```
dcrctl:
        Start            End              Type  Open Ref GrpRef Name
        0000000000400000 0000000000cee000 exe   2    0   0      dcrctl
        00000002f4d3d000 00000002f4d4a000 rlib  0    1   0      /usr/lib/libpthread.so.26.1
        00000002eeb12000 00000002eec06000 rlib  0    1   0      /usr/lib/libc.so.95.1
        00000002d4e43000 00000002d4e43000 ld.so 0    1   0      /usr/libexec/ld.so
```

Static executables used to be possible with GOFLAGS="-tags=netgo -ldflags=-extldflags=-static", but this may have changed in Go 1.13.

Setting CGO_ENABLED=0 from the OpenBSD host results in an identical executable to the one built from macOS.

**Comment by jrick** [comment]
Tested the opposite case as well (CGO_ENABLED=0 build of a macOS binary) and, even though it remains dynamically linking to libSystem ([it must](https://developer.apple.com/library/archive/qa/qa1118/_index.html)), the builds are identical.

**Comment by jrick** [comment]
Closed by 6f4a05b5
